conan/tools/cmake/cmakedeps2/cmakedeps.py

```diff
@@ -292,6 +293,7 @@ def _get_host_runtime_dirs(self):
             runtime_dirs = aggregated_cppinfo.bindirs if is_win else aggregated_cppinfo.libdirs
             for d in runtime_dirs:
                 d = d.replace("\\", "/")
+                d = relativize_path(d, self._conanfile, "${CMAKE_CURRENT_LIST_DIR}")
                 existing = host_runtime_dirs.setdefault(config, [])
                 if d not in existing:
```

Comment:
Could `existing` have absolute and relative paths? I'm asking as the for loop above is doing this:

```python
            if variable_match:
                capture = variable_match.group(1)
                matches = re.findall(r'"\$<\$<CONFIG:([A-Za-z]*)>:([^>]*)>"', capture)
                for config, paths in matches:
                    host_runtime_dirs.setdefault(config, []).append(paths)
```

Reply:
The ``existing`` is reading the paths from previous runs (previous configurations Release/Debug) from file.
In the file, they should have been made relative too, so these paths should be left as they are in the file, and the match should match because they were relativized before.